### ISSUE TITLE
Run elm-format is not available in project panel

### ISSUE THREAD

**Issue by pravdomil** [klazuka/intellij-elm#600]
Is possible to run elm-format on directory or selected files in project panel?

**prettier**
<img width="436" alt="image" src="https://user-images.githubusercontent.com/2387356/72548376-d6e40280-388e-11ea-81f4-ad4472e9d5b3.png">

**elm-format**
<img width="472" alt="image" src="https://user-images.githubusercontent.com/2387356/72548288-adc37200-388e-11ea-946e-323c39f3528c.png">

**Comment by ajalt** [comment]
Yes. Select your files, right click, then select External tools > elm-format

**Comment by pravdomil** [comment]
I can't find it?

<img width="346" alt="image" src="https://user-images.githubusercontent.com/2387356/72561916-87abcb00-38aa-11ea-95cb-d0cf1c68ffc5.png">

**Comment by ajalt** [comment]
Ah, it looks like you would need to configure the external tool manually. It should be possible for this plugin to provide the action in the menu, though.

**Comment by klazuka** [comment]
@pravdomil here are some old instructions for registering `elm-format` as an External Tool in IntelliJ: https://github.com/klazuka/intellij-elm/blob/5704b8af2791eb7e0f5d125d9878a777dc14989f/docs/elm-format/setup.md

**Comment by pravdomil** [comment]
thanks, I would be great to have it right in the actions
